regtests/setup.sh

```diff
@@ -111,7 +111,6 @@ spark.driver.extraJavaOptions -Dderby.system.home=/tmp/derby
 spark.sql.catalog.polaris=org.apache.iceberg.spark.SparkCatalog
 spark.sql.catalog.polaris.type=rest
 spark.sql.catalog.polaris.uri=http://${POLARIS_HOST:-localhost}:8181/api/catalog
-spark.sql.catalog.polaris.warehouse=snowflake
```

Comment:
The mechanism in here for "automatically" updating a local conf is to update line 95 and line 103 to increment the version number, so that the next time someone runs the regtests they'll automatically pick up the newest set of config values.

However, there's the risk that if someone manually updated the spark-defaults file with customizations then the update will comment out all such customizations.

I guess it's a balance between having a consistent conf for everyone running the tests vs allowing people to have persistent differences.

If people only run via docker then the point is moot though, so maybe the whole auto-updating mechanic in here can be considered obsolete.

Reply:
For non-docker use case, it makes sense to update the version. I can make the follow-up change.